index.html

```diff
@@ -376,5 +390,7 @@ <h2>Contact me</h2>
         <input class="btn" type="submit" value="Get In Touch" />
       </form>
     </footer>
+
+    <script src="/script.js"></script>
```

Comment:
- [ ] [OPTIONAL] When I run your project locally, the menu button works fine. However, what I noticed is that you will encounter some issues when you push changes to the deployed version. It will not work on the deployed version. This is mostly due to how you reference files in your HTML codes. With that being said, I would suggest that you update the source attribute here to `src="./script.js"` I think it might be good 😃 👍

Reply:
Thank you @BertrandConxy I have put a . before my js src file Now. Thanks for passing my review without the dot Thank you so much 👍✔